packages/app-mobile/components/screens/Note.tsx

```diff
@@ -1172,7 +1173,7 @@ class NoteScreenComponent extends BaseScreenComponent {
 		const titleComp = (
 			<View style={titleContainerStyle}>
 				{isTodo && <Checkbox style={this.styles().checkbox} checked={!!Number(note.todo_completed)} onChange={this.todoCheckbox_change} />}
-				<TextInput
+				<NoLagTextInput
```

Comment:
there's probably no need to change this, i doubt the title can get long enough to cause any lag

Reply:
actually there's just one text input that was causing issues, all others are things like tag name, dropbox token input  or encryption password which should be fine as they were

Reply:
Hm. didn't notice this point, but this is indeed true. Only note body textinput required to be replaced. Should i open another pr and close this one or you will merge by modifying?

Reply:
you can update your pr, there's nothing stopping you
just commit more changes to the same branch and github will pick the changes

Reply:
Done